### PR TITLE
Code refactor...

### PR DESCRIPTION
**Pull request by rilango**

### CODE REVIEW

File: nvidia/cheminformatics/data/helper/chembldata.py [hunk 1]
```diff
@@ -12,17 +11,17 @@
 from contextlib import closing
 from nvidia.cheminformatics.utils.singleton import Singleton
 from nvidia.cheminformatics.smiles import RemoveSalt, PreprocessSmiles
-from nvidia.cheminformatics.fingerprint import MorganFingerprint, Embeddings
+from nvidia.cheminformatics.fingerprint import Embeddings
```
Comment (mlgill) [inline nvidia/cheminformatics/data/helper/chembldata.py:14]: I think the MorganFingerprint class needs to be imported but the Embedding class does not, correct?

File: nvidia/cheminformatics/data/helper/chembldata.py [hunk 2]
```diff
@@ -12,17 +11,17 @@
...
 
 SMILES_TRANSFORMS = [RemoveSalt(), PreprocessSmiles()]
 FINGERPRINT_SELECTION = Embeddings
```
Comment (mlgill) [inline nvidia/cheminformatics/data/helper/chembldata.py:17]: Fingerprint selection should be changed to MorganFingerprint to use that by default

File: nvidia/cheminformatics/utils/fileio.py [hunk 3]
```diff
@@ -31,6 +36,56 @@ def initialize_logfile(benchmark_file=BENCHMARK_FILE):
             fh.write(f'# {config_message}\n')
             fh.write('date,benchmark_type,step,time(hh:mm:ss.ms),n_molecules,n_workers,metric_name,metric_value\n')
 
+class MaticsLogger(object):
```
Comment (mlgill) [inline nvidia/cheminformatics/utils/fileio.py:39]: I think this should be MetricsLogger, correct?

**Review by mlgill — CHANGES_REQUESTED** [review]
@rilango Have confirmed the benchmark and integration tests run for me on this code. Have made minor requests for code changes -- you can merge when addressed (or ignore them if they're in error). Thanks!

File: nvidia/cheminformatics/utils/fileio.py (hunk 3 above)
Comment (rilango) [reply nvidia/cheminformatics/utils/fileio.py:39]: > I think this should be MetricsLogger, correct?

Yes, indeed.

File: nvidia/cheminformatics/data/helper/chembldata.py (hunk 1 above)
Comment (rilango) [reply nvidia/cheminformatics/data/helper/chembldata.py:14]: Embeddings is set as default. That requires importing it. The caller will import MorgenFingerprint, to change the default. Therefore this code does not require importing MorgenFingerprint.

File: nvidia/cheminformatics/data/helper/chembldata.py (hunk 2 above)
Comment (rilango) [reply nvidia/cheminformatics/data/helper/chembldata.py:17]: This changes my previous comment

Comment (mlgill) [reply nvidia/cheminformatics/data/helper/chembldata.py:17]: @rilango My understanding is then that Embeddings were the default? Regardless, I trust you to fix the issue if it does exist. You can make any needed changes (or not) and merge the code when you're ready. You should have merge rights.

Comment (rilango) [reply nvidia/cheminformatics/data/helper/chembldata.py]: @mlgill Please check this new commit to cleanup this part of code further https://github.com/NVIDIA/cheminformatics/pull/17/commits/dd6e710b82b03af3babe52bb89a67499d4763d64

This commit also had changes to use properties file to change configuration.

**Pull request merged**